src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        <UserInfo post={post} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+    <div>
+      <CommentList comments={post.comments} />
```

Comment:
The 'CommentList' component is wrapped in a div without any styling or class name. Consider removing the div if it's not needed.